lib/auth/clt.go

```diff
@@ -471,6 +471,56 @@ func (c *Client) OktaClient() services.Okta {
 	return c.oktaClient
 }
 
+// CreatePlugins creates a new plugin resource
+func (c *Client) CreatePlugin(ctx context.Context, plugin types.Plugin) error {
+	return c.APIClient.CreatePlugin(ctx, plugin)
+}
+
+// DeleteAllPlugins deletes all plugin resources
+func (c *Client) DeleteAllPlugins(ctx context.Context) error {
+	return c.APIClient.DeleteAllPlugins(ctx)
+}
+
+// DeletePlugin deletes a plugin resource
+func (c *Client) DeletePlugin(ctx context.Context, name string) error {
+	return c.APIClient.DeletePlugin(ctx, name)
+}
+
+// GetPlugin gets a plugin resource
+func (c *Client) GetPlugin(ctx context.Context, name string, withSecrets bool) (types.Plugin, error) {
+	return c.APIClient.GetPlugin(ctx, name, withSecrets)
+}
+
+// GetPlugins gets all plugin resources
+func (c *Client) GetPlugins(ctx context.Context, withSecrets bool) ([]types.Plugin, error) {
+	plugins, err := c.APIClient.GetPlugins(ctx, withSecrets)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	retPlugins := make([]types.Plugin, 0, len(plugins.Plugins))
+	for _, plugin := range plugins.Plugins {
+		retPlugins = append(retPlugins, plugin)
+	}
+	return retPlugins, nil
+}
+
+// ListPlugins returns a list of plugin resources
+func (c *Client) ListPlugins(ctx context.Context, limit int, startKey string, withSecrets bool) ([]types.Plugin, string, error) {
```

Comment:
Just to ensure I understood this
If `withSecrets == true` then it returns the oauth client id/secret for the plugin
However, given we are not implementing SetPluginCredentials, it means that even if we set `withSecrets` to `true` it won't bring anything. Is that it?